src/jit/compiler.cpp

```diff
@@ -7225,8 +7249,8 @@ void JitTimer::PrintCsvHeader()
 
         fprintf(fp, "\"Total Cycles\",");
         fprintf(fp, "\"CPS\"\n");
-        fclose(fp);
     }
+    fclose(fp);
```

Comment:
That's what happens when you try to use notepad to apply changes (to avoid the white space issue); glad you caught it, will fix. And yes, _waccess isn't available outside of Windows (well it is, with a slightly different name, but it doesn't appear in PAL).